backends/ebpf/psa/externs/ebpfPsaRegister.cpp

```diff
@@ -0,0 +1,230 @@
+/*
+Copyright 2022-present Orange
+Copyright 2022-present Open Networking Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "ebpfPsaRegister.h"
+#include "backends/ebpf/psa/ebpfPsaControl.h"
+
+namespace EBPF {
+
+EBPFRegisterPSA::EBPFRegisterPSA(const EBPFProgram *program,
```

Comment:
This stuff could be useful even in the basic ebpf backend, but perhaps it's too late to move it there.

Reply:
We can think about moving it to EBPF model, but we would need to agree on the extern definition, which might be different from the one from PSA.